libs/onchain/src/getSigner.ts

```diff
@@ -0,0 +1,15 @@
+/* istanbul ignore file */
+import { Signer, Wallet } from "ethers"
+import getProvider from "./getProvider"
+import { Network } from "./types"
+
+export default async function getSigner(network: Network): Promise<Signer> {
+    if (!process.env["BACKEND_PRIVATE_KEY"]) {
+        throw new Error("Please set your INFURA_API_KEY in your .env file")
```

Comment:
```suggestion
        throw new Error("Please set your BACKEND_PRIVATE_KEY in your .env file")
```
A small typo in the error message, but I have fixed in my PR